unit-tests/src/test/java/org/eclipse/collections/impl/map/mutable/UnifiedMapOverridesTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2018 Goldman Sachs.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl.map.mutable;
+
+import java.util.Map;
+
+import org.eclipse.collections.api.map.MutableMap;
+
+public class UnifiedMapOverridesTest extends UnifiedMapTest
+{
+    public static class UnifiedMapOverrides<K, V> extends UnifiedMap<K, V>
+    {
+        public UnifiedMapOverrides()
+        {
+        }
+
+        UnifiedMapOverrides(int initialCapacity, float loadFactor)
+        {
+            super(initialCapacity, loadFactor);
+        }
+
+        UnifiedMapOverrides(Map<? extends K, ? extends V> map)
```

Comment:
@motlin The only reason that one constructor is public is because it's invoked using reflection. It does stick out like a sore thumb so I'll make the rest of them public too.